Cannot build on linux

Hello,

I'm trying to build this on Linux and I have compile errors:

First `u_short` doesn't exist for me, had to rename these to `unsigned short` manually. Then:

```
picoquic/picosocks.c: In function ‘picoquic_recvmsg’:       
picoquic/picosocks.c:255:81: error: dereferencing pointer to incomplete type                                                                         
                     ((struct sockaddr_in*)addr_dest)->sin_addr.s_addr = pPktInfo->ipi_addr.s_addr;                                                                               
```

--- a/picoquic/picosocks.c
+++ b/picoquic/picosocks.c
@@ -36,13 +36,13 @@ static int bind_to_port(SOCKET_TYPE fd, int af, int port)
 #else
         s4->sin_family = af;
 #endif
-        s4->sin_port = htons((u_short)port);
+        s4->sin_port = htons((unsigned short)port);
         addr_length = sizeof(struct sockaddr_in);
     } else {
         struct sockaddr_in6* s6 = (struct sockaddr_in6*)&sa;
 
         s6->sin6_family = AF_INET6;
-        s6->sin6_port = htons((u_short)port);
+        s6->sin6_port = htons((unsigned short)port);
         addr_length = sizeof(struct sockaddr_in6);
     }
 
@@ -619,12 +619,12 @@ int picoquic_get_server_address(const char* ip_address_text, int server_port,
     if (inet_pton(AF_INET, ip_address_text, &ipv4_dest->sin_addr) == 1) {
         /* Valid IPv4 address */
         ipv4_dest->sin_family = AF_INET;
-        ipv4_dest->sin_port = htons((u_short)server_port);
+        ipv4_dest->sin_port = htons((unsigned short)server_port);
         *server_addr_length = sizeof(struct sockaddr_in);
     } else if (inet_pton(AF_INET6, ip_address_text, &ipv6_dest->sin6_addr) == 1) {
         /* Valid IPv6 address */
         ipv6_dest->sin6_family = AF_INET6;
-        ipv6_dest->sin6_port = htons((u_short)server_port);
+        ipv6_dest->sin6_port = htons((unsigned short)server_port);
         *server_addr_length = sizeof(struct sockaddr_in6);
     } else {
         /* Server is described by name. Do a lookup for the IP address,
@@ -646,7 +646,7 @@ int picoquic_get_server_address(const char* ip_address_text, int server_port,
             switch (result->ai_family) {
             case AF_INET:
                 ipv4_dest->sin_family = AF_INET;
-                ipv4_dest->sin_port = htons((u_short)server_port);
+                ipv4_dest->sin_port = htons((unsigned short)server_port);
 #ifdef _WINDOWS
                 ipv4_dest->sin_addr.S_un.S_addr = ((struct sockaddr_in*)result->ai_addr)->sin_addr.S_un.S_addr;
 #else
@@ -656,7 +656,7 @@ int picoquic_get_server_address(const char* ip_address_text, int server_port,
                 break;
             case AF_INET6:
                 ipv6_dest->sin6_family = AF_INET6;
-                ipv6_dest->sin6_port = htons((u_short)server_port);
+                ipv6_dest->sin6_port = htons((unsigned short)server_port);
                 memcpy(&ipv6_dest->sin6_addr,
                     &((struct sockaddr_in6*)result->ai_addr)->sin6_addr,
                     sizeof(ipv6_dest->sin6_addr));

